scripts/bash.source

```diff
@@ -1,3 +1,9 @@
 export PATH="${REPOSITORY_ROOT}/scripts:$PATH"
 export PATH="${REPOSITORY_ROOT}/.circleci/scripts:$PATH"
-export MERGED_JUNIT_RESULT_FILE="$REPOSITORY_ROOT/merged_junit.xml"
\ No newline at end of file
+export MERGED_JUNIT_RESULT_FILE="$REPOSITORY_ROOT/merged_junit.xml"
+
+export CURRENT_COUNTRY=$(curl 'https://ipinfo.io/country')
```

Comment:
I don't know this API! Can we use this free? Not limit counts? 👀

Reply:
free, and the call limit is 1000 times per day from the same IP. So it can be assumed as no limit seemingly.